src/zenml/deployers/containerized_deployer.py

```diff
@@ -38,7 +38,9 @@
 class ContainerizedDeployer(BaseDeployer, ABC):
     """Base class for all containerized deployers."""
 
-    CONTAINER_REQUIREMENTS: List[str] = []
+    # TODO: this needs to come from the deployment settings or from the
```

Comment:
TODO

Reply:
I implemented a simple flavor class system to collect the software dependencies at runtime